src/_data/capability-matrix.yml

```diff
@@ -127,7 +146,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: "Using Spark's broadcast variables. In streaming mode, side inputs may update but only between micro-batches."
-
+          - class: apex
+            l1: 'Yes'
+            l2: size restrictions
+            l3: No distributed implementation and therefore size restrictions. Not able to push lookups directly up into key-based sources. 
```

Comment:
I believe pretty much all runners are unable to push lookups directly into key-based sources - actually the architecture to make this possible is not fully in place. You can leave the comment if you want, but it makes it sound like ApexRunner has special restrictions, but I think it is pretty much the same status as others.